app/views/organizations/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<% title @organization.name %>
+<% title sanitize @organization.name %>
```

Comment:
this we're calling two methods, we should probably use parenthesis at least in the inner one:

`title sanitize(@organization.name)`

it's easier to read IMO